generator/transit_generator.cpp

```diff
@@ -111,17 +128,71 @@ struct IsSortedVisitor
   bool m_isSorted = true;
 };
 
-/// \returns ref to a stop at |stops| by |stopId|.
-Stop const & FindStopById(vector<Stop> const & stops, StopId stopId)
+template <typename T>
+void Append(vector<T> const & src, vector<T> & dst)
+{
+  dst.insert(dst.end(), src.begin(), src.end());
+}
+
+void LoadBorders(string const & dir, string const & countryId, vector<m2::RegionD> & borders)
```

Comment:
Why not to pass CountryFile? Raw string country id is ambiguous (with or without "mwm" suffix?).

Reply:
Country id (TCountryId type) is always a string without "mwm" suffix. But here we have std::string type instead. 

I'm not sure that passing a instance of CountryFile is the best solution here. To do so we need to create a CountryFile instance on the caller side (BuildTransit() method) and then take from the instance on the callee side (LoadBorders()) country id string. 

TCountryId is declared in storage but country ids are used outside storage in a lot of places in our project. Probably it's worth moving TCountryId declaration to upper level (BTW I don't know where) and use this type for countryId values. 

As for the current place I propose to use here (in generator) TCountryId type instead of string.

What do you thank about that?